src/audio_worklet_node.rs

```diff
@@ -247,6 +308,31 @@ fn process_audio_worklet(env: &Env, processors: &JsObject, args: ProcessorArgume
 
             match res {
                 Ok(js_ret) => {
+                    // Grab back new owned value processor and js_ouputs, has been
+                    // consumed by `apply` call
+                    let processor = processors.get_named_property::<JsObject>(&id.to_string())?;
+                    let js_outputs =
+                        processor.get_property::<JsSymbol, JsObject>(k_worklet_outputs)?;
+
+                    // copy JS output buffers back into outputs
+                    for (output_number, output) in outputs.iter().enumerate() {
+                        let js_output = js_outputs.get_element::<JsObject>(output_number as u32)?;
+
+                        for (channel_number, channel) in output.iter().enumerate() {
+                            let js_channel =
+                                js_output.get_element::<JsTypedArray>(channel_number as u32)?;
+                            let js_channel_value = js_channel.into_value()?;
+                            let js_channel_buffer: &[f32] = js_channel_value.as_ref();
+
+                            let src = js_channel_buffer.as_ptr();
+                            let dst = channel.as_ptr() as *mut f32;
+
+                            unsafe {
```

Comment:
We can just safely use `copy_from_slice`, no?

Reply:
That was my first try but I did manage to find a way to `iter_mut()` the `&'static [&'static [&'static [f32]]]` output...